libsolidity/codegen/ir/IRGenerationContext.cpp

```diff
@@ -170,6 +170,42 @@ void IRGenerationContext::internalFunctionAccessed(Expression const& _expression
 	}
 }
 
+pair<u256, unsigned> IRGenerationContext::storageLocationOfStateVariable(VariableDeclaration const& _varDecl) const
+{
+	solAssert(isStateVariable(_varDecl), "");
+	return m_stateVariables.at(&_varDecl);
+}
+
+pair<string, string> IRGenerationContext::storageLocationOfLocalVariable(VariableDeclaration const& _varDecl) const
+{
+	solAssert(isLocalVariable(_varDecl), "");
```

Comment:
This is also redundant if the function is only called from https://github.com/ethereum/solidity/pull/9652/files#diff-f841efc1479453da2e05b159a7f1da6bR199